documentation/configuring_bolt.md

```diff
@@ -89,14 +85,14 @@ fields](bolt_inventory_reference.md).
 
 Use this level to set configuration that should apply to all projects for a
 particular user. Options that you might set at the user-level include paths to
-private keys, credentials for a plugin,
-or default inventory configuration that is common to all of your projects.
-You can set most configurable options in Bolt at the user level.
+private keys, credentials for a plugin, or default inventory configuration that
+is common to all of your projects. You can set most configurable options in Bolt
+at the user level.
 
-The preferred method for setting user-level configuration is to use a
-`bolt-defaults.yaml` file. This file does not allow you to set project-specific
-configuration, such as the path to an inventory file, and is less likely
-to lead to errors where Bolt loads content from another project.
+To set configuration at the user level, use a `bolt-defaults.yaml` file. This
+file does not allow you to set project-specific configuration, such as the path
+to an inventory file, and is less likely to lead to errors where Bolt loads
```

Comment:
```suggestion
file does not allow you to set project-specific configuration, such as the modulepath,
and is less likely to lead to errors where Bolt loads
```

3.0 gets rid of the `inventoryfile` config option

Reply:
Ah right, I thought I'd removed this already